src/main/nsis/samples/conf/process-conf.xml

```diff
@@ -14,7 +14,7 @@
                 <!-- password below has been encrypted using key file, therefore it will not work without the key setting: process.encryptionKeyFile
                 the password is not a valid encrypted value, please generate the real value using encrypt.bat utility -->
                 <entry key="sfdc.password" value="5555555555555555"/>
-                <entry key="process.encryptionKeyFile" value="c:\dataloader\samples\conf\sample.key"/>
+                <entry key="process.encryptionKeyFile" value="c:\users\username\sample.key"/>
```

Comment:
I recommend suggesting a subdirectory for the keyfile, such as `AppData\Local\SalesforceDataLoader\sample.key`.

Reply:
This should presumably, at least, be changed to the new default keyfile path...